build/templates/grpc_session_options.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from enum import IntEnum
+
+
+class SessionInitializationBehavior(IntEnum):
+    UNSPECIFIED = 0
+    r'''
+    It is acceptable to either initialize a new session or attach to an existing one.
+    '''
+    INITIALIZE_NEW = 1
```

Comment:
I question the value of `_NEW` and `_TO_EXISTING` in these enum values. Do they add anything?

Reply:
Improved per our discussion.